travispy/entities/hook.py

```diff
@@ -30,3 +31,29 @@ class Hook(Entity):
         'private',
         'admin',
     ]
+
+    def _set_hook(self, flag):
```

Comment:
In theory you can ask for active repos using command ``travispy.repos(active=True)`` as you can see [here](https://docs.travis-ci.com/api#repositories) however the other way around demands more code: you should retrieve all repos and then filter by the inactives.

``Hook`` will be removed once I start porting TravisPy to new Travis API and there the mentioned problem is properly solved :smile: